spec/support/fixtures/pdf-text-all-no-answers.txt

```diff
@@ -1,83 +1,82 @@
-Outbound vehicle
-Registration                   Cell number                      PECS Barcode
+Outboundvehicle
```

Comment:
⚠️ This issue with the spacing needs to be sorted before we're able to merge this PR, otherwise each new PR will potentially end up going backwards and forwards with spaces and no spaces.